netsuite/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+
+NetSuite's [SuiteTalk SOAP API](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml) provides capability to access NetSuite operations related different kind of NetSuite records
+such as Account, Client, Transactions, Invoice, Classifications etc.
+
+This module supports [NetSuite WSDL 2020.2.0](https://system.netsuite.com/help/helpcenter/en_US/srbrowser/Browser2020_2/schema/record/account.html) version.
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* A NetSuite account
+* Obtaining tokens
+1. Go to [NetSuite](https://www.netsuite.com) and login to your account.
+2. If you have NetSuite Permission to Create Integration APP, The following steps may be helpful or else go to step 03.
+    1. Enable the SuiteTalk Web service features of the account (Setup->Company->Enable Features).
+    
+    2. Create an integration application (Setup->Integration->New), enable TBA code grant and scope, and obtain the 
+    following credentials: 
+        * Client ID
+        * Client Secret
+3. If you have Client ID, Client Secret from your administrator, Obtain the below credentials by following the token based authorization in the [NetSuite documentation](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml). 
+    * Access Token
+    * Access Token Secret
+4. Obtain the SuiteTalk Base URL, which contains the account ID under the company URLs (Setup->Company->Company
+    Information).
+
+        E.g:  https://<ACCOUNT_ID>.suitetalk.api.netsuite.com
+
+## Quickstart
+To use the NetSuite connector in your Ballerina application, update the .bal file as follows:
+### Step 1 - Import connector
+* Create a Ballerina file and import the following Netsuite module and the others. 
+```ballerina
+import ballerinax/netsuite;
+```
+### Step 2 - Create a new connector instance
+2. Create a NetSuite Client by providing credentials. Initialize the connector by giving authentication details in the HTTP client config, which has built-in support for Token Based Authentication(TBA). NetSuite uses TBA to authenticate and authorize requests. The NetSuite connector can be initialized by the HTTP client config using the client ID, client secret, access token and access token secret. You may use Ballerina [Configuration variable](https://ballerina.io/learn/by-example/configurable.html) to store your credentials.
```

Comment:
Since Step 2 does not have multiple substeps, there is no need to number this.

Reply:
Suggestion: "Create a NetSuite client by providing credentials. Initialize the connector by giving authentication details in the HTTP client configuration, which has built-in support for Token Based Authentication(TBA). The NetSuite connector can be initialized by the HTTP client configuration using the client ID, client secret, access token, and the access token secret. You can use the Ballerina [Configuration variable](https://ballerina.io/learn/by-example/configurable.html) to store your credentials."